ci/tests/richards.c

```diff
@@ -1,11 +1,17 @@
 /*  C version of the systems programming language benchmark
+**  The original benchmark code is in public domain. Accordingly 
```

Comment:
s/in public/in the public/

Reply:
Fixed in [f7e847d](https://github.com/capablevms/bdwgc/pull/18/commits/f7e847d2f505b14a1a91debbcb029dc3de233831)